client/state/activity-log/utils.js

```diff
@@ -3,22 +3,45 @@
 export const filterStateToApiQuery = filter =>
 	Object.assign(
 		{},
-		{ number: 1000 },
-		filter.group && filter.group.includes && { group: filter.group.includes }
+		filter.action && { action: filter.action },
+		filter.after && { after: filter.after },
+		filter.before && { before: filter.before },
+		filter.by && { by: filter.by },
+		filter.date_range && { date_range: filter.date_range },
```

Comment:
we shouldn't have `date_range` anywhere in Calypso except at the boundaries as we use camelCasing as a project. is this function taking the internal filter state as input or the parameters from the search query?